### ISSUE TITLE
offline-plugin

### ISSUE THREAD

**Issue by NekR** [TailorDev/monod#364]
Hey, do you think you still need to use custom version of `offline-plugin`?

**Comment by willdurand** [comment]
Hi @NekR, I dont know actually. I had to tweak it a while ago to fit our needs, but now... I should look at your latest version maybe.

**Comment by NekR** [comment]
@willdurand yeah, probably, a lot of improvements there. But hey, if it works fine for you as it's then it's fine too :-)
